src/components/Error404.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import imgerror from '../assets/error404img.jpg';
+import '../components/Error404_style.css';
+
+const Error404 = () => {
+
+    const redirectToHomePage = () => {
+        window.location.href = '/'; // Replace with the desired home page URL or path
+      };
+
```

Comment:
This redirection works differently in react-router, which changes the URL without refreshing the whole page and shows Home page.